src/components/ciPipeline/ciPipeline.service.ts

```diff
@@ -294,11 +294,16 @@ export function deleteCIPipeline(
     webhookConditionList,
 ) {
     const ci = createCIPatchRequest(ciPipeline, formData, isExternalCI, webhookConditionList)
+
+    const updatedCI = {
```

Comment:
instead of fetching from ci, can we directly use `ciPipeline.id` and `formData.name`?